compose.js

```diff
@@ -89,7 +89,11 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
     if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    dstDescriptor.initializers.push.apply(dstDescriptor.initializers, srcDescriptor.initializers.filter(isFunction));
+    srcDescriptor.initializers.forEach(i => {
```

Comment:
Yea I guess it's very subjective, I even think that the line with `if (!Array.isArray(dstDescriptor.initializers))` is very hard to spot in there and it's almost unclear at a first glance why it's there. With reduce you clearly see how it's used and handled. But I am not going to fight you on this, if others like your way better too, it's fine.